cli/zone.go

```diff
@@ -93,7 +92,7 @@ func runGetZone(cmd *cobra.Command, args []string) {
 		log.Errorf("Object %d: no zone config found", id)
 		return
 	}
-	fmt.Fprintln(osStdout, rows[0][1])
+	fmt.Println(rows[0][1])
```

Comment:
the examples output comparator replaces stdout with a pipe:
```
src/testing/example.go:
// Capture stdout.
    50		stdout := os.Stdout
    51		r, w, err := os.Pipe()
    52		if err != nil {
    53			fmt.Fprintln(os.Stderr, err)
    54			os.Exit(1)
    55		}
    56		os.Stdout = w
```

Whereas we do the following in cli.go:
```
var osStdout = io.Writer(os.Stdout)
```

This means that we'll be bypassing the pipe, making the output invisible to the example runner.